### ISSUE TITLE
Knothide quiver and ammo pouch recipes

### ISSUE THREAD

**Issue by Grimfeather** [ZhengPeiRu21/mod-individual-progression#234]
https://i.gyazo.com/d8a7665984cc471354bc5be619401e0e.jpg

Noticed this at the LW trainer in Shattrath.
These 2 recipes require lvl71. which of course is weird for a TBC recipe.
This character of mine is on progression tier 6, the first one in TBC.
I did find out that this recipe was not available at launch in TBC. So I'm not really sure what the solution here is.
According to wowhead comments this recipe was added in patch 2.3 which I believe is progression tier 10. (Zul'Aman)
https://www.wowhead.com/tbc/item=34099/knothide-ammo-pouch#comments

Something is wrong here though.
As an example, glove reinforcements is currently available at the LW trainer and does not require lvl 71 to learn.
https://www.wowhead.com/tbc/spell=44770/glove-reinforcements#recipes
And this was also added in patch 2.3.

-------------

I found this in professions.sql

-- Patch 2.3 added easy to get catch-up ammo pouches
-- Allow these to remain available for completionists, but make them require having passed TBC progression
UPDATE `npc_trainer` SET `ReqLevel` = 71 WHERE `SpellID` IN (44343, 44344);

So the lvl 71 requirement was intentional.

-------------

adding the lvl 71 requirement would not have been my solution here, but if that's here to stay then I guess we need to add this requirement to other recipes added in patch 2.3 as well.

from the 2.3 patch:
https://wowwiki-archive.fandom.com/wiki/Patch_2.3.0

- glove reinforcements
- the 20 slot quivers and ammo pouches (which now already have the lvl71 requirement)
- quiver of a thousand feathers (24 slots) from lower city 
- netherscale ammo pouch from thrallmar / honor hold
- leatherworker's satchel (20 slot LW bag)

-------------

I would prefer to see the lvl71 requirement removed though. 
And if possible add the progression tier 10 requirement to these instead.
If that's not possible I would still prefer to remove the lvl 71 requirement, it's just really weird to see this at the trainer.

**Comment by ZhengPeiRu21** [comment]
Thanks for the write-up. It is an ugly solution, but intentional. Unfortunately there currently isn't a good way to add a progression requirement to trainers, so this was my work-around to prevent skipping progressions and getting powerful crafted gear that makes more complicated progression paths obsolete. Perhaps a better work-around would be to create an alternate version of the trainer that is phased to only appear at the correct progression phase - I'll keep this issue open and post updates here.

**Comment by Grimfeather** [comment]
I been trying to fix this today. But I run into an issue.
I did what ZhengPei suggested. I added an alternate version of the trainer.
I phase that copy out with script: npc_ipp_tbc_pre_t4
I phase the original trainer in with: npc_ipp_tbc_t4
That works correctly.

The problem is, I can't get the alternate trainer to actually offer training.
I can't figure out why the training menu won't open.

If someone can point me in the right direction to find the solution, that would be great.
My changes to Darmari, the LW trainer, are at the bottom of this file:
https://github.com/Grimfeather/mod-individual-progression/blob/TBC-limited-LW-recipes-before-TBC-T4/sql/world/base/professions.sql

plus a single line in ipp_aware_npcs.sql
